docs/internals/contributing/committing-code.txt

```diff
@@ -29,58 +29,60 @@ to standard themselves.
 
 .. _Jenkins wiki page: https://code.djangoproject.com/wiki/Jenkins
 
-Here is one way to commit a pull request::
+An easy way to checkout a pull request locally is to use the following script
+(``upstream`` is assumed to be ``django/django``):
 
-    # Create a new branch tracking upstream/master -- upstream is assumed
-    # to be django/django.
-    git checkout -b pull_xxxxx upstream/master
+.. code-block:: bash
 
-    # Download the patches from github and apply them.
-    curl https://github.com/django/django/pull/xxxxx.patch | git am
+    #!/bin/sh
+    git fetch upstream pull/$1/head:pr/$1
```

Comment:
This can be added as an alias in .gitconfig:

```ini
pr = !sh -c \"git fetch upstream pull/${1}/head:pr/${1} && git checkout pr/${1}\" -
```

https://github.com/berkerpeksag/dotfiles/blob/master/gitconfig#L17